app/controllers/teams_controller.rb

```diff
@@ -14,9 +14,16 @@ def index
   # GET /teams/1
   # GET /teams/1.json
   def show
+    raise ActiveRecord::RecordNotFound if @team.name.starts_with?("portus_global_team_")
```

Comment:
Nitpicking: maybe add an empty line afterwards, to differentiate better this case with the rest of the "block" :)